Src/AutoFixture.NUnit3/FixedNameTestCaseParameterBuilder.cs

```diff
@@ -0,0 +1,100 @@
+﻿
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using NUnit.Framework.Interfaces;
+using NUnit.Framework.Internal;
+using System.Globalization;
+
+namespace Ploeh.AutoFixture.NUnit3
+{
+    /// <summary>
+    /// Builder that generates fixed values for the <see cref="TestCaseParameters"/>.OriginalArguments
```

Comment:
Rephrase doc to emphasize that this strategy generate static test name and omit the implementation details.